scripts/cnv_wdl/germline/cnv_germline_case_scattered_workflow.wdl

```diff
@@ -0,0 +1,231 @@
+#
+# A wrapper for gCNV case workflow intended for lowering computing cost by making it feasible to use 
+# preemptible cloud instances with low memory requirements. CPU, memory and disc requirements can be 
+# lowered for GermlineCNVCaller and DetermineGermlineContigPloidy tasks. 
+# 
+#
+# - Example invocation:
+#
+#       java -jar cromwell.jar run cnv_germline_case_scattered_workflow.wdl -i my_parameters.json
+#
+####################
+
+import "cnv_germline_case_workflow.wdl" as GermlineCNVCaseWorkflow
+
+workflow CNVGermlineCaseScatteredWorkflow {
+
+    ##################################
+    #### required basic arguments ####
+    ##################################
+    File intervals
+    Array[String]+ normal_bams
+    Array[String]+ normal_bais
+    File contig_ploidy_model_tar
+    Array[File]+ gcnv_model_tars
+    Int num_intervals_per_scatter
+    File ref_fasta_dict
+    File ref_fasta_fai
+    File ref_fasta
+    String gatk_docker
+    Int num_samples_per_scatter_block = 25 #recommended value for WES
```

Comment:
Hmm, I've lost track of how we are supposed to specify default parameter values in WDLs.   Having these default values here seems idiosyncratic compared to the other CNV WDLs.  Can you not push this down to the task level?  Perhaps @LeeTL1220 can comment.

Reply:
It seems more consistent to keep `num_samples_per_scatter_block` as required with no default (since this parameter is pretty fundamental to this WDL), and specify the default for `preemptible_attempts` in the task using `select_first` as we do elsewhere.  You've given the defaults for these parameters in the documentation, and we can eventually provide them in the default json for FireCloud, so I think we don't need to set them here.

Reply:
I agree that's better. Left `num_samples_per_scatter_block` with no default and pushed down default for `preemptible_attempts` into `select_first` of each task